db/db_impl.cc

```diff
@@ -1625,7 +1625,7 @@ bool DBImpl::HasActiveSnapshotLaterThanSN(SequenceNumber sn) {
   if (snapshots_.empty()) {
     return false;
   }
-  return (snapshots_.newest()->GetSequenceNumber() > sn);
+  return (snapshots_.newest()->GetSequenceNumber() >= sn);
```

Comment:
This should be ">", isn't it? 
">=" has a different meaning than what the function name (`HasActiveSnapshotLaterThanSN`) implies.

Reply:
A snapshot takes the last sequence number written. So I can argue a snapshot with the same sequence number is still "later than" the sequence. The method is only used by blob db and this is a quick fix to the sequence number checking. I'll make a proper fix later this week. See the added TODO in this patch.

Reply:
Ah, ok. Didn't know that its only used by BlobDB now. Sounds good.